src/sentry/static/sentry/app/utils/requestError/requestError.tsx

```diff
@@ -1,4 +1,5 @@
 export default class RequestError extends Error {
+  responseText?: any;
```

Comment:
shouldn't this type be of type `string | null` ?

Reply:
I updated this to be `responseText?: string` because it does not get set to `null` anywhere.